e2e/rbd.go

```diff
@@ -1297,6 +1297,67 @@ var _ = Describe("RBD", func() {
 				validateRBDImageCount(f, 0)
 			})
 
+			By("create a thick-provisioned PVC", func() {
+				err := deleteResource(rbdExamplePath + "storageclass.yaml")
+				if err != nil {
+					e2elog.Failf("failed to delete storageclass with error %v", err)
+				}
+				err = createRBDStorageClass(f.ClientSet, f, nil, map[string]string{
+					"thickProvision": "true"}, deletePolicy)
+				if err != nil {
+					e2elog.Failf("failed to create storageclass with error %v", err)
+				}
+
+				pvc, err := loadPVC(pvcPath)
+				if err != nil {
+					e2elog.Failf("failed to load PVC with error %v", err)
+				}
+				pvc.Namespace = f.UniqueName
+
+				err = createPVCAndvalidatePV(f.ClientSet, pvc, deployTimeout)
+				if err != nil {
+					e2elog.Failf("failed to create PVC with error %v", err)
+				}
+				validateRBDImageCount(f, 1)
```

Comment:
once we create the pvc can we validate the storage consumption size in the ceph cluster? is that what we are doing with `rbd diff` if yes can we compare the allocated size with the expected size.

Reply:
I think you can only check the storage usage of an RBD image with `rbd diff`. It is possible to count the size of all the extents together and verify that the size of the image matches. But, this goes a little far, I think, it would be sufficient to validate that _some extents_ have been allocated.

Reply:
checking `ceph df --format=json` for the pool help to identify checking the used size before and after creating the image?

Reply:
That assumes there are no other things happening on the pool, I do not think we can assume exclusive access to it.

Reply:
or even `rbd du` command helps?

Reply:
I'll use `rbd du` for the final version of this PR.